src/Controller/QuestionTemplateController.php

```diff
@@ -82,12 +82,16 @@ public function deleteQuestion(Request $request, array $requestAttributes): Resp
 
     public function getQuestions(Request $request, array $requestAttributes): Response
     {
-        $count = $this->questionTemplateService->getQuestionNumber();
+        $filters = [];
+        if (isset($requestAttributes['text'])) {
+            $filters['text'] = $requestAttributes['text'];
+        }
+        $count = $this->questionTemplateService->getQuestionNumber($filters);
```

Comment:
can be changed to
`$filters = isset($requesAttributes['text']) ? ['text' => $requesAttributes['text']] : [];`